src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -313,6 +313,67 @@ impl ArithmeticOp for DecimalMultiply {
     }
 }
 
+#[rpn_fn]
+#[inline]
+fn abs_int(arg: &Option<Int>) -> Result<Option<Int>> {
+    match arg {
+        Some(arg) => {
+            if *arg == std::i64::MIN {
```

Comment:
You could use `checked_abs` to avoid spelling out MIN explicitly. Also that `.to_owned` shouldn't be necessary